openwisp_notifications/notification_types.py

```diff
@@ -0,0 +1,78 @@
+from django.core.exceptions import ImproperlyConfigured
```

Comment:
please shorten this file to `types`, imports will then look like `from openwisp_notifications.types import register_notification_type`.